scripts/main/lychee.js

```diff
@@ -425,55 +467,69 @@ lychee.login = function (data) {
  * @returns {void}
  */
 lychee.loginDialog = function () {
-	// Make background unfocusable
-	tabindex.makeUnfocusable(header.dom());
-	tabindex.makeUnfocusable(lychee.content);
-	tabindex.makeUnfocusable(lychee.imageview);
-
-	const msg = lychee.html`
-			<a class='signInKeyLess' id='signInKeyLess'>${build.iconic("key")}</a>
-			<form>
-				<p class='signIn'>
-					<input class='text' name='username' autocomplete='on' type='text' placeholder='$${
-						lychee.locale["USERNAME"]
-					}' autocapitalize='off' data-tabindex='${tabindex.get_next_tab_index()}'>
-					<input class='text' name='password' autocomplete='current-password' type='password' placeholder='$${
-						lychee.locale["PASSWORD"]
-					}' data-tabindex='${tabindex.get_next_tab_index()}'>
-				</p>
-				<p class='version'>Lychee ${lychee.version}<span> &#8211; <a target='_blank' href='${lychee.updateURL}' data-tabindex='-1'>${
-		lychee.locale["UPDATE_AVAILABLE"]
-	}</a><span></p>
-			</form>
-			`;
+	const loginDialogBody = `
+		<a id='signInKeyLess' class="button"><svg class='iconic'><use xlink:href='#key'/></svg></a>
+		<form class="force-first-child">
+			<div class="input-group stacked">
+				<input class='text' name='username' autocomplete='username' type='text' autocapitalize='off' data-tabindex='${tabindex.get_next_tab_index()}'>
+			</div>
+			<div class="input-group stacked">
+				<input class='text' name='password' autocomplete='current-password' type='password' data-tabindex='${tabindex.get_next_tab_index()}'>
+			</div>
+		</form>
+		<p class='version'>Lychee <span class='version-number'></span><span class="update-status up-to-date"> &#8211; <a target='_blank' href='${
```

Comment:
Same question about the need for `up-to-date` here as before.

Reply:
See previous answer.

Reply:
I removed the unnecessary `else`-branch in https://github.com/LycheeOrg/Lychee-front/commit/8279248d69de6ec8e4a7df7f597cf302c25e3cb2.